services/configServices/__tests__/NetlifyTomlService.js

```diff
@@ -0,0 +1,54 @@
+const axios = require("axios")
+const { Base64 } = require("js-base64")
+
+const validateStatus = require("@utils/axios-utils")
+
+const {
+  netlifyTomlContent,
+  netlifyTomlHeaderValues,
+} = require("@root/fixtures/netlifyToml")
+
+const { GITHUB_BUILD_ORG_NAME, GITHUB_BUILD_REPO_NAME } = process.env
+
+jest.mock("axios")
+
+describe("Auth Service", () => {
```

Comment:
i think you might have meant `NetlifyToml Service`!

Reply:
[a6cdec7](https://github.com/isomerpages/isomercms-backend/pull/357/commits/a6cdec7b64e02ef1769b77c35a2cfcfa3b882986)